<launch>
  <node name="formation0" output="screen" pkg="formation_controller" type="formation_controller_node">
    <param name="agent_id" value="uav0" />
    <param name="uav_id" value="0" />
    <param name="x_offset" value="0.0" />
    <param name="y_offset" value="0.0" />
    <param name="z_offset" value="0" />
    <param name="control_mode" value="1" />
  </node>
  <node name="formation1" output="screen" pkg="formation_controller" type="formation_controller_node">
    <param name="agent_id" value="uav1" />
    <param name="uav_id" value="1" />
    <param name="x_offset" value="4.992285935" />
    <param name="y_offset" value="0.27763491" />
    <param name="z_offset" value="0" />
    <param name="control_mode" value="1" />
  </node>
  <node name="formation2" output="screen" pkg="formation_controller" type="formation_controller_node">
    <param name="agent_id" value="uav2" />
    <param name="uav_id" value="2" />
    <param name="x_offset" value="-0.27763491" />
    <param name="y_offset" value="4.992285935" />
    <param name="z_offset" value="0" />
    <param name="control_mode" value="0" />
  </node>
  <node name="formation3" output="screen" pkg="formation_controller" type="formation_controller_node">
    <param name="agent_id" value="uav3" />
    <param name="uav_id" value="3" />
    <param name="x_offset" value="-4.992285935" />
    <param name="y_offset" value="-0.27763491" />
    <param name="z_offset" value="0" />
    <param name="control_mode" value="0" />
  </node>
  <node name="formation4" output="screen" pkg="formation_controller" type="formation_controller_node">
    <param name="agent_id" value="uav4" />
    <param name="uav_id" value="4" />
    <param name="x_offset" value="0.27763491" />
    <param name="y_offset" value="-4.992285935" />
    <param name="z_offset" value="0" />
    <param name="control_mode" value="0" />
  </node>
</launch>